frontend/src/app/grant.service.ts

```diff
@@ -16,7 +16,7 @@ export class GrantService {
 
   constructor(private http: HttpClient) {}
 
-  getGrants(): Observable<Grant[]> {
-    return this.http.get<Grant[]>(this.grantsUrl);
+  getGrants(): Observable<any> {
```

Comment:
**nitpick/issue (transient)**: (I can't tell whether this is a nitpick or an issue, but either way I think it might be temporary.) In order to access the `data` key in the JSON response, we're removing the strong typing (`Grant[]`) on this request, and mapping the data over in [grants.component.ts#L22](https://github.com/USDAForestService/NRM-Grants-Agreements/pull/403/files#diff-239bbb376128f567352822d9694c065d29a9e67e7740a32476d19f472fbde4cbR22).

The variable `grants: Grant[]` in grants.component.ts is still strongly typed, so we're still typing it. I don't think this introduces a risk, but I wanted to flag it for attention.

Also, if we implement [a JSON:API library](https://jsonapi.org/implementations/#client-libraries-typescript), this issue will go away, hence the "transient" in the label.

Reply:
**question (non-blocking):** I wonder if we should typdef the response that includes the data member with a value of `Grant[]` which is essentially what a JSON:API library would do for us?

Reply:
@neilmb Are you comfortable with us leaving this as a todo? I included a comment with direction for looking into this further. I'm hesitant to look into more typedeffing for this because 1) we're 100% sure it works right now, 2) the methods I can think of may take time away from transition sprint work, and 3) implementing a JSON:API client might obviate needing to do anything here.

Reply:
Yes, I'm fine keeping this as the TODO.